.ci/openshift-ci/build_install.sh

```diff
@@ -19,12 +19,68 @@ source /etc/os-release
 
 source "${cidir}/lib.sh"
 
-[ -z "$1" ] && die "Usage: $0 path/to/install/dir"
+usage() {
+	cat <<-EOF
+	Usage: $0 path/to/install/dir
+
+	This script is used by the CI job on OpenShift CI to build and install
+	Kata Containers in a given directory.
+
+	Environment variables:
+
+	- Use SANDBOXED_CONTAINERS_CONF="yes" to configure like the OpenShift
+	  Sandboxed Containers.
+	EOF
+}
+
+if [ -z "$1" ]; then
+	usage
+	die "$0: missing parameter"
+fi
+
 export DESTDIR="$1"
 info "Build and install Kata Containers at ${DESTDIR}"
 
 [ "$(id -u)" -ne 0 ] && die "$0 must be executed by privileged user"
 
+# This applies a set of build configurations akin to OpenShift
+# Sandboxed Containers.
+#
+# The following environment variables have the origin on the Fedora rawhide
+# specfile [*].
+#
+# [*] https://src.fedoraproject.org/rpms/kata-containers/blob/rawhide/f/kata-containers.spec
+sandboxed_containers_build_configs() {
+	export KERNELTYPE="compressed"
+	export DEFSHAREDFS="virtio-fs"
+	export DEFVIRTIOFSCACHESIZE=0
+	export DEFSANDBOXCGROUPONLY=true
+	export MACHINETYPE="q35"
+	export FEATURE_SELINUX="yes"
+	export DEFENABLEANNOTATIONS=['\".*\"']
+}
+
+# This ensures the guest image is built to work with the host Kernel akin to
+# OpenShift Sandboxed Containers.
+sandboxed_containers_image_configs() {
+	# It needs to overwrite some variables of
+	# kata-containers/tools/osbuilder/Makefile.
+	export DRACUT_KVERSION="4.18.0-305.19.1.el8_4.x86_64"
+	export DRACUT_CONF_DIR=$(mktemp -d --tmpdir dracut.conf.d.XXXX)
+	yum install -y "kernel-modules-${DRACUT_KVERSION}"
+	# Unlike the kernel installed by the .ci/install_kata_kernel.sh script,
+	# the host kernel doesn't have the needed modules built static. Thus,
+	# we need to package them as loadable modules into the guest image.
+	cp -r "${kata_repo_dir}/tools/osbuilder/dracut/dracut.conf.d"/* \
+		"${DRACUT_CONF_DIR}"
+	cat <<-EOF >> "${DRACUT_CONF_DIR}/10-drivers.conf"
+	drivers+="vmw_vsock_virtio_transport virtio_net virtiofs "
+	drivers+="virtio_blk virtio_scsi virtio_console "
+	drivers+="vfio-pci vfio vfio_iommu_type1 irqbypass vfio_virqfd "
```

Comment:
It would spread the code over extra lines, but it might be better to add one driver per line. That makes diffs easier for future changes and also emphasises that crucial space at the end of each of these lines. We can also then sort the list alphabetically ;)

Reply:
@jodh-intel I agree. I will change that.

Reply:
Done.